### PR TITLE
feat(React Incubation): User Profile

### PR DESCRIPTION
**Pull request by gabe-lyons**
Currently using mock data. Integration w/ graphql coming soon.

Screen Shot 2021-02-04 at 3 38 04 PM

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by jjoyce0510 — APPROVED** [review]
overall this lgtm. excited to see this tied into the GQL api

File: datahub-web-react/src/conf/Global.tsx [hunk 1]
```diff
@@ -14,5 +14,4 @@ export enum PageRoutes {
     BROWSE = '/browse',
     BROWSE_RESULTS = '/browse/:type',
     DATASETS = '/datasets',
-    USERS = '/users',
```
[the remark this answers is not in the document]
Comment (gabe-lyons) [reply datahub-web-react/src/conf/Global.tsx:17]: @jjoyce0510 

the route is now being generated here:

https://github.com/linkedin/datahub/pull/2083/files/57fb739b1e6399a2d01d0401743631e20b6bd31e#diff-da3b2a9eb0f0bbc3ed5273e88bd873bb9c9f44b27cd2ab6716e951cc310d79c3R39

and the entity pathname is now coming from the entity registry:

https://github.com/linkedin/datahub/pull/2083/files/57fb739b1e6399a2d01d0401743631e20b6bd31e#diff-da3b2a9eb0f0bbc3ed5273e88bd873bb9c9f44b27cd2ab6716e951cc310d79c3R39

File: .github/workflows/build-and-test.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ on:
   pull_request:
     branches:
       - master
+      - react_graphql_feature_branch
```
Comment (shirshanka) [inline .github/workflows/build-and-test.yml:12]: This should be removed

Comment (gabe-lyons) [reply .github/workflows/build-and-test.yml]: @shirshanka good catch- removed

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**